consumer/pubsub/src/main/java/com/spotify/heroic/consumer/pubsub/Connection.java

```diff
@@ -53,7 +53,7 @@
     private final ConsumerSchema.Consumer consumer;
     private final long maxOutstandingElementCount;
     private final long maxOutstandingRequestBytes;
-    private Subscriber subscriber = null;
```

Comment:
can you make this `final` now to remain consistent with the other variables?

Reply:
:+1:

Reply:
It can't be final since it isn't set in the constructor.